src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -71,6 +71,7 @@ def start_service(args):
     os.environ[PF_NO_INTERACTIVE_LOGIN] = "true"
     port = args.port
     get_app()
+    app.config["DEBUG"] = args.debug
```

Comment:
I think flask will enable code listening and auto reload in this case?

Reply:
updated, won't take effect when create app in a new detach process in start_service code. Use environment variable instead.